docs/development/build-instructions-windows.md

```diff
@@ -8,11 +8,22 @@ Follow the guidelines below for building Electron on Windows.
 * Visual Studio 2017 15.7.2 or higher - [download VS 2017 Community Edition for
   free](https://www.visualstudio.com/vs/)
 * [Python 2.7](http://www.python.org/download/releases/2.7/)
+  * Note, that you will need at least version 2.7.10. Right now `depot_tools`
+  still comes with version 2.7.6, which will cause the `gclient` command to fail
+  (see https://crbug.com/868864). Other than explained in the GN build
+  instructions linked below, you have to configure your **PATH** so that your
+  locally installed Python comes before the `depot_tools` folder.
 * [Node.js](https://nodejs.org/download/)
 * [Git](http://git-scm.com)
 * [Debugging Tools for Windows](https://msdn.microsoft.com/en-us/library/windows/hardware/ff551063.aspx)
   if you plan on creating a full distribution since `symstore.exe` is used for
   creating a symbol store from `.pdb` files.
+  * If you run into build problems using the latest Windows SDK, try to install
```

Comment:
There is an exact version of the SDK that Chromium is known to build against and we have on our CI machines, that version should be documented here cc @jkleinsc

Reply:
On [Checking out and Building Chromium for Windows](https://chromium.googlesource.com/chromium/src/+/master/docs/windows_build_instructions.md#visual-studio) they say that Windows SDK version `10.0.17134` is needed. I am using the older version `10.0.15063.468` which seems to work, too.

Reply:
@MartinMa That doc is for Chromium master, Electron master is back on Chromium 67 and might have different reqs

Reply:
> I am using the older version 10.0.15063.468 which seems to work, too.

Docs for Chromium 67 mention exactly `10.0.15063`.

https://chromium.googlesource.com/chromium/src/+/67.0.3396.127/docs/windows_build_instructions.md#visual-studio